@xen-orchestra/web/src/components/site/dashboard/Backups.vue

```diff
@@ -53,4 +56,28 @@ const segments = computed<DonutChartWithLegendProps['segments']>(() => [
     color: 'disabled',
   },
 ])
+
+const vmsProtectionTitle = computed<DonutChartWithLegendProps['title']>(() => ({
+  label: t('backups.vms-protection'),
+  iconTooltip: t('backups.vms-protection.tooltip'),
+  icon: faCircleInfo,
+}))
+
+const vmsProtectionSegments = computed<DonutChartWithLegendProps['segments']>(() => [
+  {
+    label: t('backups.vms-protection.protected'),
+    value: record.value?.backups?.vmsProtection?.protected ?? 0,
```

Comment:
I'm not a fan of the idea of ​​having a default value if `backups` is `undefined`, the data displayed in the dashboard will be wrong.
In Figma, I don't see any `Something goes wrong` visual like we have for XOLite. Do we have any information on this?

Reply:
Indeed, we need to improve how we handle such cases (no data vs errors, etc.).
For now I can use the same `NoData` component I made for the `BackupIssues` component on the dashboard, it would look like this if `backups` is `undefined`:

![image](https://github.com/user-attachments/assets/ce760636-b7e4-4333-99a6-e597716679ea)